src/components/common/PostalCode/PostalCodeInput.tsx

```diff
@@ -1,21 +1,22 @@
-import React, { useState } from 'react'
+import { useStorage } from '@faststore/sdk'
 import { Input as UIInput, Label as UILabel } from '@faststore/ui'
-import type { ChangeEvent, KeyboardEvent } from 'react'
-import usePostalCode from 'src/hooks/usePostalCode'
+import React, { useRef } from 'react'
+import type { KeyboardEvent } from 'react'
+
 import './postal-code-input.scss'
 
+const POSTAL_CODE_STORAGE_KEY = 'main::store::postalCode'
 const POSTAL_CODE_INPUT_ID = 'postal-code-input'
 
 export default function PostalCodeInput() {
-  const [localPostalCode, setLocalPostalCode] = useState<string>('')
-  const [, setPostalCode] = usePostalCode()
-  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
-    setLocalPostalCode(event.target.value)
-  }
+  const [code, setPostalCode] = useStorage<string>(POSTAL_CODE_STORAGE_KEY, '')
```

Comment:
Just to better indicate that these variables are related, what do you think of renaming `code` to `postalCode` or `setPostalCode` to `setCode`?

Reply:
This hook was planned to be used in other components/hooks. But we was doing the feature step by step